### ISSUE TITLE
Responsible AI Service blocks response to "yay it works" ¯\_(ツ)_/¯ 

### ISSUE THREAD

**Issue by pauleboutin** [microsoft/vscode-copilot-release#1965]
Type: <b>Bug</b>

the Responsible AI Service is far too zealous. 

Extension version: 0.21.2
VS Code version: Code 1.94.2 (384ff7382de624fb94dbaf6da11977bba1ecd427, 2024-10-09T16:08:44.566Z)
OS version: Darwin arm64 24.0.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.28GB free)|
|Process Argv|--crash-reporter-id 4f4233e0-5ea4-4d31-8c22-693554002d97|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
vscaat:30438848
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementsc:30995553
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
f3je6385:31013174
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
nativerepl2:31139839
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-t:31132770
nativeloc1:31134641
wkspc-ranged-t:31151552
cf971741:31144450
autoexpandse:31146404
iacca2:31156134
notype1:31157159
5fd0e150:31155592
dwcopilot:31162478
iconenabled:31158251

```

</details>

**Comment by sprivite** [comment]
I am also getting blocked by Responsible AI Service for extremely benign queries. My query:

"The date selection is incorrect. You need to handle this in interpreting the graph."

**Comment by bhavyaus** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have: https://github.com/microsoft/vscode-copilot-release/issues/2625. Please refer to that one for updates and discussions. Feel free to open a new issue if you think this is a different problem.
